packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget/main/index.md

```diff
@@ -80,3 +84,85 @@ When you upgrade an embedded Sign-In Widget:
+
+* `features.webauthn` &mdash; Prevented the Widget from invoking the legacy Windows Hello factor. This feature is no longer configurable in the Widget. See the [sign-on policy](https://help.okta.com/oie/en-us/Content/Topics/identity-engine/policies/about-okta-sign-on-policies.htm) in the Okta product documentation to enable and configure a possession authenticator.
+
+* `features.selfServiceUnlock` &mdash; Displayed the "Unlock Account" link so that users could unlock their accounts. This feature is no longer configurable in the Widget. See [self-service account recovery](https://help.okta.com/oie/en-us/Content/Topics/identity-engine/authenticators/configure-sspr.htm) to enable this feature.
+
+* `features.multiOptionalFactorEnroll` &mdash; Allowed users to enroll in multiple optional factors before they finished the authentication flow. This feature is no longer configurable in the Widget. The [MFA enrollment and rules](https://help.okta.com/oie/en-us/Content/Topics/identity-engine/policies/about-mfa-enrollment-policies.htm) will be evaluated by default when applicable.
+
+* `features.registration` &mdash; Displayed the registration section in the primary auth page. This feature is no longer configurable in the Widget. The [profile enrollment policies](https://help.okta.com/oie/en-us/Content/Topics/identity-engine/policies/create-profile-enrollment-policy-sr.htm) will be evaluated by default when applicable.
+
+* `features.idpDiscovery` &mdash; Enabled IdP Discovery when the user signed in. This feature is no longer configurable in the Widget. The [Identity Provider routing rules](https://help.okta.com/oie/en-us/Content/Topics/Security/Identity_Provider_Discovery.htm) will be evaluated by default when applicable.
+
+* `features.showPasswordToggleOnSignInPage` &mdash; Provided end users with the ability to toggle visibility of their password on the Okta Sign-In page so that they could check their password entry before clicking **Sign In**, and prevented an account lock-out due to exceeding their org's permitted number of failed sign-in attempts. This feature is no longer configurable in the Widget. It may be added to policy configuration in a future release.
+
+* `features.scrollOnError` &mdash; Scrolled errors into view. This feature is no longer configurable in the Widget. Errors are going to be displayed inside the Widget.
+
+* `features.skipIdpFactorVerificationBtn` &mdash; Provided an automatic redirect to the selected Identity Provider when selected from the list of factors. This feature is no longer configurable in the Widget. It is no longer needed since the optional authenticators have the ability to skip by default.
+
+### I18n properties
+
+After you upgrade your org to Okta Identity Engine, you can override existing text strings in the interface with Identity Engine i18n strings so that you can create localized Widgets. See [Updates to the Sign-In Widget i18n properties](/docs/guides/oie-upgrade-sign-in-widget-i18n).
```

Comment:
```suggestion
After you upgrade your org to the Identity Engine, you can override existing text strings in the interface with Identity Engine i18n strings so that you can create localized Widgets. See [Updates to the Sign-In Widget i18n properties](/docs/guides/oie-upgrade-sign-in-widget-i18n).
```